dropgz/build/cniTest_windows.Dockerfile

```diff
@@ -1,30 +0,0 @@
-ARG ARCH

```

Comment:
Yay, I'm excited to just be able to use `azure-cni` as an init container, so cool! 🤩